### PR TITLE
release-->main

### PR DESCRIPTION
**Pull request by Raj-StepSecurity**

### CODE REVIEW

File: .github/workflows/check-mirrors.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (shubham-stepsecurity) [inline .github/workflows/check-mirrors.yaml:1]: This workflow can be removed, It is used to validate mirror & send notification if something breaks!
view here: https://github.com/step-security/agent-api/pull/2696/files#diff-af9ad324e06f7ea5cec44a1110ebc060be501470a2aadc2254b0013238e77044

Also remove `check-mirrors.sh` as well

File: main.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs').promises;
+const core = require('@actions/core');
+const tc = require('@actions/tool-cache');
```
Comment (shubham-stepsecurity) [inline main.js:5]: import `axios` here: (const axios = require('axios');)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@actions/cache": "^3.2.4",
+    "@actions/core": "^1.10.1",
+    "@actions/exec": "^1.1.1",
+    "@actions/github": "^6.0.0",
+    "@actions/tool-cache": "^2.0.1",
+    "libsodium-wrappers": "^0.7.15"
```
Comment (shubham-stepsecurity) [inline package.json:19]: add `axios` dependency here

File: main.js (hunk 2 above)
Comment (Raj-StepSecurity) [reply main.js:5]: done

File: package.json (hunk 3 above)
Comment (Raj-StepSecurity) [reply package.json:20]: done

File: .github/workflows/check-mirrors.yaml (hunk 1 above)
Comment (Raj-StepSecurity) [reply .github/workflows/check-mirrors.yaml]: done

**Comment by shubham-stepsecurity** [comment]
LGTM

**Review by shubham-stepsecurity — APPROVED** [review]

**Pull request merged**